cache/models/samples.js

```diff
@@ -0,0 +1,155 @@
+/**
```

Comment:
I can try keeping the api level stuff like logging, extracting info from req object and response in api layer only. And then just do the redis transactions here and return the result object from here.